Vagrantfile

```diff
@@ -18,9 +18,14 @@ Vagrant.configure(2) do |config|
         echo "America/Detroit" > /etc/timezone
         dpkg-reconfigure -f noninteractive tzdata
 
+        debconf-set-selections <<< 'mysql-server mysql-server/root_password password MySuperPassword'
+        debconf-set-selections <<< 'mysql-server mysql-server/root_password_again password MySuperPassword'
+
```

Comment:
Also, since these password settings go along with the `apt-get install -y mysql-server` line that appears later, why not put them together so it's clear that they're related?